src/firefox/index.js

```diff
@@ -310,8 +310,12 @@ export type InstallExtensionParams = {
  * text file that contains the path to the extension source.
  */
 export function installExtension(
-  {asProxy=false, manifestData, profile, extensionPath}: InstallExtensionParams
-): Promise<any> {
+  {
+    asProxy = false,
+    manifestData,
+    profile,
+    extensionPath,
+  }: InstallExtensionParams): Promise<any> {
```

Comment:
I autofixed everything with `$ ./node_modules/.bin/eslint {tasks,tests,src,*.js} --fix`, but this was the one place where the _new_ line length now exceeded 80 and therefore threw an error.